zjsonpatch/src/main/java/io/fabric8/zjsonpatch/JsonDiff.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (C) 2015 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.fabric8.zjsonpatch;
+
+import com.fasterxml.jackson.databind.JsonNode;
+
+public class JsonDiff {
+  private JsonDiff() {
+  }
+
+  public static JsonNode asJson(final JsonNode source, final JsonNode target) {
+    return com.flipkart.zjsonpatch.JsonDiff.asJson(source, target);
```

Comment:
I had to reintroduce this class due to https://github.com/fabric8io/kubernetes-client/pull/6276#issuecomment-2293363660